src/main/java/hudson/plugins/git/Revision.java

```diff
@@ -21,8 +21,8 @@
 public class Revision implements java.io.Serializable, Cloneable {
     private static final long serialVersionUID = -7203898556389073882L;
 
-    ObjectId sha1;
-    Collection<Branch> branches;
+    private ObjectId sha1;
+    private Collection<Branch> branches;
```

Comment:
I believe that this changes the serialization of the Revision object without changing its serialVersionUID.  Changing serialization has been a source of problems in the past.  I'd much rather that we not change serialization unless there are compelling reasons that benefit users of the git client plugin.  

This pull request provides a behavior preserving transformation (a refactoring).  A refactoring benefits maintainers of the git client plugin.  That is a desirable thing, but it is not sufficient to justify the extra effort to confirm that the change of serialization is worth the effort.  Changing serialization risks that this would not be a behavior preserving transformation, since the change of serialization may actually cause a behavior change.
```suggestion
    ObjectId sha1;
    Collection<Branch> branches;
```

Reply:
Ohh Ok, @MarkEWaite. I didn't think about serialization compatibility. However, as per the coding standard, making those variables private as part of the OOP's encapsulation would be better for the file. 

Can we change the serialization number? Or is it not possible? will you please tell me that?

Reply:
> Can we change the serialization number? Or is it not possible? will you please tell me that?

Certainly possible, but it needs much more testing than I am willing to apply to this pull request.  A change of serialization then requires testing to confirm that the previously serialized objects are correctly handled by the new code.

Changing serialization in Jenkins plugins has been a source of repeated bugs for me.  I'm not willing to risk another serialization issue for a code improvement that is more about style than about improving things for users.  There are many style flaws in the plugin that I accept rather than risking damage to a user in order to improve style.

Reply:
Ohh, you are right. I'd rather make the suggested changes.